phys/module_diag_solar.F

```diff
@@ -39,6 +39,7 @@ SUBROUTINE solar_diag (rho, dz8w, ph, phb, cldfrac3d, coszen, swdnb, swdnt, qv,
 !----------------------------------------------------------------------
 
      USE module_model_constants, ONLY: G
+     USE module_state_description, ONLY: param_first_scalar, p_qc, p_qi, p_qs
```

Comment:
@twjuliano @dudhia @weiwangncar 
Tim,
Would it be better to have these come through an argument list. That would fit better with an eventual paradigm we are heading towards.

Reply:
@davegill 
Dave, this has been fixed.